bin/crew

```diff
@@ -1099,21 +1099,23 @@ end
 
 def expand_dependencies
   def push_dependencies
-    if @pkg.is_binary?(@device[:architecture]) ||
-      (!@pkg.in_upgrade && !@pkg.build_from_source && @device[:installed_packages].any? { |pkg| pkg[:name] == @pkg.name })
-      # retrieve name of dependencies that doesn't contain :build tag
-      check_deps = @pkg.dependencies.select {|k, v| !v.include?(:build)}.map {|k, v| k}
-    else
-      # retrieve name of all dependencies
-      check_deps = @pkg.dependencies.map {|k, v| k}
-    end
-    # check all dependencies recursively
-    check_deps.each do |dep|
-      # build unique dependencies list
-      unless @dependencies&.include?(dep) || dep == @pkgName
-        @dependencies << dep
-        search dep, true
-        push_dependencies
+    unless @pkg.nil?
```

Comment:
or just `return if @pkg.nil?`
or even `return unless @pkg`

and then the whole block below doesn't have to be indented